docs/open-source-project/release-notes/tizen-5-0-m1.md

```diff
@@ -0,0 +1,952 @@
+
+​		**o**   **Fixed D-bus policies**
+
+​		**o**   **Fixed memory leak & potential defects**
+
+​	**·**        **Maps-plugin-here**
+
+​		**o**   **Fixed D-bus policies**
+
+​		**o**   **Fixed memory leak & potential defects.**
+
+​	**·**        **Sensor**
+
+​		**o**   **Fixed Memory leaks & potenatil defects**
+
+​	**·**        **Email**
+
+​		**o**   **Fixed memory leak & potential defects**
+
+​	**·**        **Message-Service**
+
+​		**o**   **Fixed memory leaks & potential defects**
+
+​	**·**        **Context**
+
+​		**o**   **Fixed memory leaks & potential defects**
+
+​	**·**        **Calendar**
+
+​		**o**   **Fixed memory leaks & potential defects**
+
+​	**·**        **Location** 
+
+​		**o**   **Fixed memory leaks & potential defects**
+
+##  
+
+## **Web Framework**
+
+ **New and Changed Features**
+
+​	**·**        **Web Engine**
+
+​		**o**   **Applying latest open source chromium(blink) : from M56 to M63**
+
+​			**§**  **Provide new W3C Standard  (Animated PNG, Full featured Indexed DB, CSS grid layout, Web USB, MSE/EME)**
+
+​			**§**  **Enhancement standard web features (WebRTC, ARIA 1.1)**
+
+​			**§**  **Web performance enhancement (V8 : ES 6 performance enhancement, Web Assembly default enabling)**
+
+ 
+
+## **Tizen .NET**
+
+ **New and Changed Features**
+
+​	**·**        **.NET Runtime**
+
+​		**o**   **Memory optmization patches have been merged.**
+
+​	**·**        **.NET Launcher**
+
+​		**o**   **Plugin interface for dll preloading is added.**
+
+​	**·**        **Xamarin.Forms**
+
+​		**o**   **Latest Xamarin.Forms stable version (3.0.0) support.**
+
+​		**o**   **Xamarin.Forms nuget package (Xamarin.Forms.nupkg) supports tizen as a default platform since Xamarin.Forms 3.0.0.**
+
+​		**o**   **TargetIdiom.Watch is now available for wearable profile.**
+
+​	·        Tizen.CircularUI
+
+​		**o**   **Circular UI extension (Tizen.CircularUI) is now available.**
+
+​	**·**        **TizenFX**
+
+​		**o**   **Support for Application Filter by installed location has been added.**
+
+​		**o**   **Enumeration and method for StorageDevice has been added.**
+
+​		**o**   **Audio only mode for Player has been added.**
+
+​		**o**   **Attach Panel has been added.**
+
+​		**o**   **Support for multiple camera and hue has been added.**
+
+​		**o**   **System settings for vibration, time update, developer option state has been added.**
+
+​		**o**   **Support for Wi-Fi AP update has been added.**
+
+​		**o**   **NUI Version has been upgraded from 0.2.66 to 0.2.83.**
+
+​		**o**   **USB Host API has been added.**
+
+​		**o**   **Tizen.Security.Privilege.GetPrivacyPrivilegeStatus() has been deprecated.**
+
+​		**o**   **360 videos support has been added in Tizen.Multimedia.Player.**
+
+**Known Issues**
+
+​	**·**        **Xamarin.Forms**
+
+​		**o**   A list of limitations is available at [here](https://developer.tizen.org/ko/development/api-reference/.net-application/current-xamarin.forms-limitations?langredirect=1)

```

Comment:
For more information on the list of limitations, see ...

Reply:
will accept